README.md

```diff
@@ -42,22 +42,56 @@ const stimuli = [{difficulty: -3, item: 'item1'}, {difficulty: -2,  item: 'item2
 const nextItem = cat.findNextItem(stimuli, 'MFI');
 ```
 
```

Comment:
more broadly, can we document and show sample code: how to set up a jsClowder, and some basic functions to run a clowder.